### ISSUE TITLE
`sby` depends on `click` without documenting this or specifying a version constraint

### ISSUE THREAD

**Issue by whitequark** [YosysHQ/sby#234]

**Comment by nakengelhardt** [comment]
I did add it to the prerequisites the other day: https://yosyshq.readthedocs.io/projects/sby/en/latest/install.html#prerequisites

**Comment by whitequark** [comment]
Ah, I see. It still does not include a version constraint though!

**Comment by nakengelhardt** [comment]
Yes, we don't have that for any of the prerequisites - it's not even settled what the minimum python version we still want to support is...

**Comment by whitequark** [comment]
Right, so I'm not missing anything then. In YoWASP I'll continue testing on 3.7+ (there are some releases that were installable on 3.6, but ubuntu-latest no longer supports that so I didn't actually test that, and now there's a ~=3.7 constraint). I'll ping you if anything breaks the CI, then.
